test/metabase/query_processor_test/offset_test.clj

```diff
@@ -310,3 +310,12 @@
                  (mt/formatted-rows
                   [->local-date 2.0]
                   (qp/process-query query)))))))))
+
+(deftest multiple-limits
+  (let [total_count (-> (mt/user-real-request :crowberto :get 200 "search?q=product")
+                        :data count)
+        result_count (-> (mt/user-real-request :crowberto :get 200 "search?q=product&limit=2&limit=3")
+                         :data count)]
+    (is (and
```

Comment:
I think it would be better to have two `is`s here, that test for each one of these seperately. That way when one or the other (or both) fail you'll know in the test report.